ayushma/models/token.py

```diff
@@ -0,0 +1,45 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from django.conf import settings
+
+from ayushma.token import RandomStringTokenGenerator
+
+
+class ResetPasswordToken(models.Model):
```

Comment:
@cp-Coder can we use `BaseModel` here? You don't need to provide an id field, everything will be managed by the class.